README.md

```diff
@@ -120,6 +120,11 @@ Non-autonomous vehicles can also be integrated with RMF provided their positions
 
 In the airport terminal map, a `Caddy` is spawned in the far right corner and can be controlled with `geometry_msgs/Twist` messages published over the `cmd_vel` topic. 
 
+Run `teleop_twist_keyboard` to control the `caddy` with your keyboard:
+```bash
+ros2 run teleop_twist_keyboard teleop_twist_keyboard
+```
```

Comment:
Nit, it seems we have lauch files for this, can you change the instructions to use those instead? (So it would also work for ignition if we used the `_ign` launch file)

Reply:
Updated the docs. Note that the previous `airport_terminal_caddy.launch.xml`  is using a physical joystick as controller input, thus it is not included here. (since users mostly use keyboard)